build_tools/docker/swiftshader/Dockerfile

```diff
@@ -9,20 +9,16 @@ WORKDIR /install-swiftshader
 
 RUN apt-get update && apt-get install -y git
 
-ARG SWIFTSHADER_COMMIT=755b78dc66b2362621a78b6964a9df3af94e960c
+ARG SWIFTSHADER_COMMIT=c27e99245d423648fe5f0eafa73d49ce7b53d201
+
+# zlib and xcb/shm.h are needed for compiling SwiftShader.
+RUN apt-get update && apt-get install -y zlib1g-dev libxcb-shm0-dev
 
-# zlib is needed for compiling SwiftShader.
-RUN apt-get update && apt-get install -y zlib1g-dev
 RUN git clone https://github.com/google/swiftshader
 RUN cd swiftshader && git checkout "${SWIFTSHADER_COMMIT?}" && cd ..
 # Only build SwiftShader Vulkan ICD.
 RUN cmake -S swiftshader/ -B build-swiftshader/ \
            -GNinja \
-           -DSWIFTSHADER_BUILD_VULKAN=ON \
```

Comment:
Have instructions for building swiftshader changed? I think we may wasn't to drop the comment about only building Vulkan if we're no longer explicitly setting anything to do that

Reply:
Some of the options no longer exist: https://swiftshader.googlesource.com/SwiftShader/+/c27e99245d423648fe5f0eafa73d49ce7b53d201/CMakeLists.txt#148, as of https://swiftshader-review.googlesource.com/c/SwiftShader/+/60688:

> Remove OpenGL ES targets from the CMake build
>
> The legacy SwiftShader OpenGL ES implementation has been deprecated for
two years, in favor of SwANGLE. Specifically the CMake build was only
meant for development and internal testing purposes, and should have no
further uses.

I was seeing warnings (not errors) for setting unknown options.

The `# Only build SwiftShader Vulkan ICD.` comment still applies to setting the `-DSWIFTSHADER_BUILD_TESTS=OFF` option and building just the `vk_swiftshader` target.